quixstreams/app.py

```diff
@@ -712,7 +785,11 @@ def add_source(self, source: BaseSource, topic: Optional[Topic] = None) -> Topic
         )
         return topic
 
-    def run(self, dataframe: Optional[StreamingDataFrame] = None):
+    def run(
+        self,
+        dataframe: Optional[StreamingDataFrame] = None,
+        timeout: Optional[float] = None,
```

Comment:
Same here, I think we can keep the default float here and ditch the `None`
```suggestion
        timeout: float = 0.0,
```